README.md

```diff
@@ -22,7 +22,11 @@ TO PROVIDE TRUE DOUBLE PRECISION CALCULATIONS. ALSO, THE INITIAL DATA
 FOR PLANETS IN "big.in" FILE WAS UPDATED (see http://ssd.jpl.nasa.gov).
 FOR ANY QUESTIONS ABOUT THIS MODIFICATION PLEASE CONTACT WITH ME USING
 THIS E-MAIL:
-idovgalyov@4xxi.com                                         2016-12-05
+idovgalyov@4xxi.com
```

Comment:
There is no need to provide an e-mail in README. It is possible to create an issue directly on github.